chaincode-labs/chaincode-residency/2018-10-22-alex-bosworth-channel-management.md

```diff
@@ -5,10 +5,9 @@ tags: ['lightning', 'routing']
 categories: ['residency']
 speakers: ['Alex Bosworth']
 date: 2018-10-22
+video: https://www.youtube.com/watch?v=HlPIB6jt6ww
```

Comment:
there should be a convention of whether we are expecting <>s or not.

Reply:
I'm planning to test how this displays in practise and report back. Because this is metadata, it makes more sense to be without <>s and to just add them when rendering the markdown.

Reply:
@tvpeter I just verified that having <>s doesn't provide help on the [Hugo side](https://github.com/bitcointranscripts/bitcointranscripts.github.io/blob/master/themes/ace-documentation/layouts/_default/single.html) when rendering the link from the parameter. You'll need to remove them.